### PR TITLE
Features/delivery page

### PR DESCRIPTION
**Pull request by Dawood-Javed**

### CODE REVIEW

**Review by haroonzafarFrizhub — APPROVED** [review]

**Review by Atif252 — CHANGES_REQUESTED** [review]
Code is not properly formatted

File: src/components/Menu/Hero.js [hunk 1]
```diff
@@ -122,21 +124,21 @@ function HeroDelivery() {
...
-  useEffect(() => {
-    fetchProductsByCategory();
-  }, []);
-
   const [menus, setMenus] = React.useState([]);
 
   const fetchDeliverableMenus = async () => {
     try {
       const res = await getDeliverableMenus();
+      console.log({ res });
```
Comment (Atif252) [inline src/components/Menu/Hero.js:132]: Remove console.log

**Review by Atif252 — CHANGES_REQUESTED** [review]

File: src/components/Menu/menuStyles.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+.Field {
+    width: 119%;
+    padding: 10px;
+    color: black;
+    font-size: 16px;
+    font-weight: 500;
+    border: 1px solid black;
+    border-radius: 10px;
+}
```
Comment (haroonzafarFrizhub) [inline src/components/Menu/menuStyles.css:32]: End of Line.

File: public/index.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/favico.ico" />
```
Comment (haroonzafarFrizhub) [inline public/index.html:5]: Rename the file correctly.

**Review by haroonzafarFrizhub — APPROVED** [review]

**Pull request merged**